tests/queue.spec.ts

```diff
@@ -150,6 +151,41 @@ describe('ARRAY LIKE USAGE', () => {
 	});
 });
 
+describe('Iteration', () => {
```

Comment:
Iteration and Iterator are slightly different. We are in fact testing iteration, but the goal of this section is to test the iterator object. 

- [ ] Change describe block label from `Iteration` to `Iterator`